Patch by Luciano Bello <luciano@linux.org.ar> for dsniff >= 2.4b1, which
adds the missing includes of arpa/inet.

--- dsniff-2.4b1/decode_aim.c		2007-06-17 16:22:39.000000000 -0300
+++ dsniff-2.4b1/decode_aim.c.inet	2007-06-17 16:26:46.000000000 -0300
@@ -14,6 +14,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <arpa/inet.h>
 
 #include "hex.h"
 #include "buf.h"
--- dsniff-2.4b1/decode_mmxp.c		2007-06-17 16:22:39.000000000 -0300
+++ dsniff-2.4b1/decode_mmxp.c.inet	2007-06-17 16:26:46.000000000 -0300
@@ -21,6 +21,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <arpa/inet.h>
 
 #include "buf.h"
 #include "decode.h"
--- dsniff-2.4b1/decode_pptp.c		2007-06-17 16:22:39.000000000 -0300
+++ dsniff-2.4b1/decode_pptp.c.inet	2007-06-17 16:26:46.000000000 -0300
@@ -16,6 +16,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <arpa/inet.h>
 
 #include "buf.h"
 #include "decode.h"
--- dsniff-2.4b1/decode_tds.c		2007-06-17 16:26:46.000000000 -0300
+++ dsniff-2.4b1/decode_tds.c.inet	2007-06-17 16:26:46.000000000 -0300
@@ -19,6 +19,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <strlcat.h>
+#include <arpa/inet.h>
 
 #include "decode.h"
 
--- dsniff-2.4b1/decode_vrrp.c		2007-06-17 16:22:39.000000000 -0300
+++ dsniff-2.4b1/decode_vrrp.c.inet	2007-06-17 16:26:46.000000000 -0300
@@ -15,6 +15,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <arpa/inet.h>
 
 #include "buf.h"
 #include "decode.h"
--- dsniff-2.4b1/ssh.c			2007-06-17 16:26:46.000000000 -0300
+++ dsniff-2.4b1/ssh.c.inet		2007-06-17 16:26:46.000000000 -0300
@@ -23,6 +23,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <arpa/inet.h>
 #include <unistd.h>
 
 #include "hex.h"
